src/controllers/chatController.ts

```diff
@@ -0,0 +1,35 @@
+import express from 'express';
+import { util } from '../config/gameInfoUtil';
+import pg from 'pg';
+import { parse } from 'pg-connection-string';
+require('dotenv').config();
+
+const dbUrl: string = process.env.DB_URL || '';
+const config: object = parse(dbUrl);
+const pool = new pg.Pool(config);
+pool.connect();
+
+export const chatController = {
+  async getChat(req: express.Request, res: express.Response) {
+    const internalGameName: string = req.params['game_name'];
+    const gameIndex: number = util.gameToIndex(internalGameName);
+    res.send(await getChatFromDB(gameIndex));
+  },
+};
+
+async function getChatFromDB(gameIndex: number) {
```

Comment:
Maybe some documentation for these methods?

Reply:
Done https://github.com/knative-portability/Kubercade/pull/16/commits/d62424954f193759c65d4deb93a78c8837949a2a